docs/src/main/sphinx/functions/list-by-topic.rst

```diff
@@ -0,0 +1,548 @@
+=========================================
+List of functions and operators per topic
+=========================================
+
+Aggregate
+---------
+
+Details in :doc:`aggregate`.
+
+- :func:`arbitrary`
+- :func:`approx_distinct`
+- :func:`approx_most_frequent`
+- :func:`approx_percentile`
+- :func:`approx_set`
+- :func:`array_agg`
+- :func:`avg`
+- :func:`bitwise_and_agg`
+- :func:`bitwise_or_agg`
+- :func:`bool_and`
+- :func:`bool_or`
+- :func:`checksum`
+- :func:`corr`
+- :func:`count`
+- :func:`count_if`
+- :func:`covar_pop`
+- :func:`covar_samp`
+- :func:`every`
+- :func:`geometric_mean`
+- :func:`histogram`
+- :func:`kurtosis`
+- :func:`map_agg`
+- :func:`map_union`
+- :func:`max`
+- :func:`max_by`
+- :func:`merge`
+- :func:`min`
+- :func:`min_by`
+- :func:`multimap_agg`
+- :func:`numeric_histogram`
+- :func:`qdigest_agg`
+- :func:`regr_intercept`
+- :func:`regr_slope`
+- :func:`skewness`
+- :func:`sum`
+- :func:`stddev`
+- :func:`stddev_pop`
+- :func:`stddev_samp`
+- :func:`tdigest`
+- :func:`variance`
+- :func:`var_pop`
+- :func:`var_samp`
+
+Array
+-----
+
+Details in :doc:`array`.
+
+- :func:`all_match`
+- :func:`any_match`
+- :func:`array_distinct`
+- :func:`array_intersect`
+- :func:`array_union`
+- :func:`array_except`
+- :func:`array_join`
+- :func:`array_max`
+- :func:`array_min`
+- :func:`array_position`
+- :func:`array_remove`
+- :func:`array_sort`
+- :func:`arrays_overlap`
+- :func:`cardinality`
+- :func:`concat`
+- :func:`combinations`
+- :func:`contains`
+- :func:`element_at`
+- :func:`filter`
+- :func:`flatten`
+- :func:`ngrams`
+- :func:`none_match`
+- :func:`reduce`
+- :func:`repeat`
+- :func:`reverse`
+- :func:`sequence`
+- :func:`shuffle`
+- :func:`slice`
+- :func:`transform`
+- :func:`zip`
+- :func:`zip_with`
+
+Binary
+------
+
+Details in :doc:`binary`
+
+The ``||`` operator performs concatenation.
+
+- :func:`concat`
```

Comment:
Function names can only link to a single definition, and all of these in binary that are also in string will link to the string version. You can see in `binary.rst` that these are marked as `:noindex:` for that reason. Make all of those non-links. (verify that all of the links are to the correct page)

Reply:
Done.